src/service/templates/models/mongoose/templates/service.json

```diff
@@ -1,7 +1,16 @@
 {
-  "require": "feathers-mongoose",
-  "options": {
-    "model": "./service.model",
+  "require": "{{answers.model.require}}",
```

Comment:
My initial intention was that we can have just 1 `service.json` template with all the common stuff and then in the actual metalsmith middleware we can read in the JSON file and manipulate it as we need. No need for handlebars.